paimon-trino-common/src/test/java/org/apache/paimon/trino/TestTrinoITCase.java

```diff
@@ -130,6 +130,23 @@ protected QueryRunner createQueryRunner() throws Exception {
             commit.commit(0, writer.prepareCommit(true, 0));
         }
 
+        {
+            Path tablePath = new Path(warehouse, "default.db/empty_t");
```

Comment:
To add a test table, we need to adapt the test cases for testCreateTable, testDropTable, and testRenameTable.

Reply:
Fixed